netctl/commands.go

```diff
@@ -292,40 +292,48 @@ var Commands = []cli.Command{
 		Usage: "router capability configuration",
 		Subcommands: []cli.Command{
 			{
-				Name:      "delete",
-				Usage:     "Delete Bgp neighbor",
-				ArgsUsage: "[router]",
+				Name:      "ls",
+				Aliases:   []string{"list"},
+				Usage:     "List BGP configuration",
+				ArgsUsage: "[hostname]",
+				Flags:     []cli.Flag{jsonFlag, quietFlag},
+				Action:    listBgp,
+			},
+			{
+				Name:      "rm",
+				Aliases:   []string{"delete"},
+				Usage:     "Delete BGP configuration",
+				ArgsUsage: "[hostname]",
+				Flags:     []cli.Flag{},
+				Action:    deleteBgp,
+			},
+			{
+				Name:      "create",
+				Usage:     "Add BGP configuration.",
+				ArgsUsage: "[hostname]",
```

Comment:
@jainvipin  yes spoke to @vvb about this . He is going to take care of it in the next commit.